data/training-modules.yml

```diff
@@ -1,20 +1,67 @@
 ---
 child-development-and-the-eyfs:
-  title: 1. Child development and the EYFS
-  thumbnail: introduction_to_supporting_physical_development
-  description: about child-development-and-the-eyfs
   duration: 1.5
+  title: Child development and the EYFS
+  thumbnail: introduction_to_supporting_physical_development
+  description: |
+    As an early years practitioner, understanding how children develop will enable
+    you to meet the needs of all the children in your care.
+  objective: |
+    This module will help you to reflect on why knowledge of child development is
+    important and help you to effectively implement the EYFS.
+    It also considers the impact of the coronavirus (COVID-19) pandemic on children.
+
+    **What you’ll learn**
+
+    On completion of this module you will be able to:
+
+    - explain the importance of early childhood education and care including strategies
+      you can use to provide a positive approach
+    - explore the importance of child development knowledge including an awareness
+      of contributing factors
+    - consider effective delivery of the EYFS
+    - describe the impact that the COVID-19 impact has had on children's development
 
```

Comment:
Is this the same information as what is on the certificate? I was wondering where to put that information.

Reply:
Will try and confirm for you